### PR TITLE
Remove use of global ert

### PR DESCRIPTION
**Pull request by oyvindeide**
## Pre review checklist

- [ ] Added appropriate labels

Adding labels helps the maintainers when writing release notes, see sections and the
corresponding labels here: https://github.com/equinor/ert/blob/main/.github/release.yml

### CODE REVIEW

File: ert_gui/gert_main.py [hunk 1]
```diff
@@ -60,15 +60,15 @@ def run_gui(args):
     # Changing current working directory means we need to update the config file to
     # be the base name of the original config
     args.config = os.path.basename(args.config)
+    # TODO: Send ErtNotifier in explicitly
```
Comment (ManInFez) [inline ert_gui/gert_main.py:63]: Make issue instead of TODO in code

File: ert_gui/simulation/ensemble_experiment_panel.py [hunk 2]
```diff
@@ -5,47 +5,51 @@
...
+from ert_shared.libres_facade import LibresFacade
+from res.enkf import EnKFMain
 from ert_gui.simulation.simulation_config_panel import SimulationConfigPanel
+from ert_shared.models import EnsembleExperiment
 
 
 class EnsembleExperimentPanel(SimulationConfigPanel):
-    def __init__(self):
-        SimulationConfigPanel.__init__(self, EnsembleExperiment)
+    def __init__(self, ert: EnKFMain, notifier):
```
Comment (ManInFez) [inline ert_gui/simulation/ensemble_experiment_panel.py:19]: Type notifies

File: ert_gui/tools/load_results/load_results_tool.py [hunk 3]
```diff
@@ -21,19 +21,20 @@
...
+        self.facade = facade
         super(LoadResultsTool, self).__init__(
             "Load results manually",
             "tools/load_manually",
             resourceIcon("ide/table_import"),
         )
         self.__import_widget = None
         self.__dialog = None
-        self.setEnabled(LoadResultsModel.isValidRunPath())
+        self.setEnabled(LoadResultsModel.isValidRunPath(self.facade._enkf_main))
```
Comment (ManInFez) [inline ert_gui/tools/load_results/load_results_tool.py:33]: accessing private member.

**Review by ManInFez — COMMENTED** [review]
Nice work. Missing typing in function signatures that have been modified. Also commit messages could be a bit more descriptive

File: ert_gui/tools/load_results/load_results_tool.py (hunk 3 above)
Comment (oyvindeide) [reply ert_gui/tools/load_results/load_results_tool.py]: It is not ideal, I know. If it is ok with your I would keep 2 of them, and make an issue to replace it completely with `LibresFacade`. The reason for keeping the 2 is then I dont have to drag `EnKFMain` deep into the GUI-tree

**Pull request closed without merging**